VotingApplication/VotingApplication.Web/Scripts/Controllers/ManagePollTypeController.js

```diff
@@ -110,17 +113,17 @@
         }
 
         function activate() {
-            ManageService.registerPollObserver(function () {
-                $scope.poll = ManageService.poll;
-                $scope.poll.MaxPerVote = $scope.poll.MaxPerVote || 3;
-                $scope.poll.MaxPoints = $scope.poll.MaxPoints || 7;
-                startingPollType = $scope.poll.PollType;
-
-                startingMaxPerVote = $scope.poll.MaxPerVote;
-                startingMaxPoints = $scope.poll.MaxPoints;
+            ManageService.getPollType($scope.manageId)
+            .then(function (data) {
+
+                $scope.MaxPerVote = data.MaxPerVote || 3;
+                $scope.MaxPoints = data.MaxPoints || 7;
+                $scope.PollType = data.PollType;
```

Comment:
`$scope.poll` isn't used anymore but still exists, and the new properties `MaxPerVote`, `MaxPoints`, and `PollType` are only defined here.